crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -446,6 +502,18 @@ where
         self.into_configs().after(set)
     }
 
+    /// Run before all systems in `set` and systems in `set` do not care about the `deferred` effects
+    /// being applied.
```

Comment:
```suggestion
    /// Run before all systems in `set`. Unlike [`before`](Self::before), this will not cause the systems in `set` to wait for the deferred effects of `self` to be applied.
```